package.json

```diff
@@ -75,13 +75,16 @@
     "countly-sdk-nodejs": "^22.6.0",
     "decompress": "^4.2.1",
     "dotenv": "^16.3.1",
-    "ethers": "^5.7.2",
+    "ethers": "6.6.0",
     "filenamify": "^6.0.0",
     "get-random-values": "^2.1.0",
     "inquirer": "^9.2.9",
     "ipfs-http-client": "^60.0.1",
     "lodash-es": "^4.17.21",
     "lokijs": "^1.5.12",
+    "multiaddr": "10.0.1",
+    "multiformats": "^12.0.1",
+    "node-fetch": "3.3.2",
```

Comment:
please remove multiaddr it's replaced by @multiformats/multiaddr
and node-fetch - it's replaced by native featch that nodejs now has